include/onnxruntime/core/common/spin_pause.h

```diff
@@ -23,6 +23,12 @@ inline void SpinPause() {
 #endif
 }
 
+inline void SpinTPAUSE() {
+#if defined(_M_AMD64) || defined(__x86_64__)
+  _tpause(0x0, __rdtsc() + 2000);
```

Comment:
Maybe add a local constant spin_delay_cycles=2000 (or using the appropriate unit) to be explicit what the number 2000 indicates

Reply:
Sure, that'll make it clearer. I'll make the change.